scripts/ci/run-test-default.sh

```diff
@@ -85,6 +85,7 @@ ${TESTCMD} --label=System.Xaml --timeout=5m make -w -C mcs/class/System.Xaml run
 ${TESTCMD} --label=System.Net.Http --timeout=5m make -w -C mcs/class/System.Net.Http run-test
 ${TESTCMD} --label=System.Json --timeout=5m make -w -C mcs/class/System.Json run-test
 ${TESTCMD} --label=System.Threading.Tasks.Dataflow --timeout=5m make -w -C mcs/class/System.Threading.Tasks.Dataflow run-test
+if [[ ${label} == linux* ]]; then ${TESTCMD} --label=System.Runtime.CompilerServices.Unsafe-xunit --timeout=5m make -w -C mcs/class/System.Runtime.CompilerServices.Unsafe run-xunit-test; fi
```

Comment:
why are we only running this on Linux?

Reply:
actually, this won't run at all since don't have `label` with value `linux` (we have `debian-xx` and `ubuntu-xx`)

Reply:
What should I use to skip on `Linux i386` only ?

Reply:
`ubuntu-1404-i386`. I assume this is due to some crash/bug on i386 we have tracked already?

Reply:
yep, https://bugzilla.xamarin.com/show_bug.cgi?id=60865

Reply:
cool, can you pleas link that in the file?